reference/import_export/simple-import.rst

```diff
@@ -0,0 +1,44 @@
+How import works
+================
+
+.. _minimal: https://github.com/akeneo/pim-community-dev/tree/master/src/Pim/Bundle/InstallerBundle/Resources/fixtures/minimal
+.. _icecat: https://github.com/akeneo/pim-community-dev/tree/master/src/Pim/Bundle/InstallerBundle/Resources/fixtures/icecat_demo_dev
+
+When you install an instance of Akeneo PIM, a set of data are imported: products, attributes, users, etc. For example, you can see the entities imported by the `minimal`_ or the `icecat`_ fixtures sets.
```

Comment:
this sentence is more related to installer, I would move it in customize installer section to explain that installer relies on import system.